python/tk_multi_publish2/processing/plugin.py

```diff
@@ -37,15 +37,23 @@ def __init__(self, path, settings, logger):
 
         self._bundle = sgtk.platform.current_bundle()
 
-        # create an instance of the hook
-        self._hook_instance = self._bundle.create_hook_instance(self._path)
-
         self._logger = logger
         self._settings = {}
 
+        # create an instance of the hook
+        self._hook_instance = self._create_hook_instance(self._path)
+
         # kick things off
         self._validate_and_resolve_config()
 
+    def _create_hook_instance(self, path):
+        """
+        Create the plugin's hook instance. Subclasses can reimplement for more
+        sophisticated hook instantiation.
+        :return:
```

Comment:
nit: looks like it's missing some docstring infos.

Reply:
addressed.